cmd/soroban-cli/src/commands/contract/deploy/utils.rs

```diff
@@ -2,14 +2,12 @@ use regex::Regex;
 
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
-    #[error(
-        "alias must be 1-30 chars long, and have only letters, numbers, underscores and dashes"
-    )]
+    #[error("alias must have only letters, numbers, underscores and dashes")]
     InvalidAliasFormat { alias: String },
 }
 
 pub fn alias_validator(alias: &str) -> Result<String, Error> {
-    let regex = Regex::new(r"^[a-zA-Z0-9_-]{1,30}$").unwrap();
+    let regex = Regex::new(r"^[a-zA-Z0-9_-]{1,}$").unwrap();
```

Comment:
I think this
```suggestion
    let regex = Regex::new(r"^[a-zA-Z0-9_-]+$").unwrap();
```
is more idiomatic regex